src/WrapperApp/components/Simulation/SimulationStatus.tsx

```diff
@@ -74,6 +74,36 @@ export default function SimulationStatus({ simulation, loadResults }: Simulation
 		loadResults?.call(null, null);
 	};
 
+	const onClickShowError = () => {
+		const errorWindow = window.open();
+
+		if (!errorWindow) return console.error('Could not open new window');
+		errorWindow.document.open();
+		errorWindow.document.write(
+			`<html>
+			<head>
+				<title>Error Details</title>
+			</head>
+			<body>
+			<h1>Details</h1>			
+			<h2>Input Files</h2>
+			<h3>beam.dat</h3>
+			<pre>${simulation?.inputFiles!['beam.dat']}</pre>
+			<h3>mat.dat</h3>
+			<pre>${simulation?.inputFiles!['mat.dat']}</pre>
+			<h3>detect.dat</h3>
+			<pre>${simulation?.inputFiles!['detect.dat']}</pre>
+			<h3>geo.dat</h3>
+			<pre>${simulation?.inputFiles!['geo.dat']}</pre>
+
+			<h2>Error</h2>
+			<pre>${simulation?.shieldhitlog}</pre>
```

Comment:
This should go in first place when error happens. Logfile can be potentially very long (>10k lines).
Is it possible to show it in a windows with a scrollbar ?

Reply:
Logfiles and input files should be displayed on separate pages.